#!/usr/bin/python3
def raise_exception():
    dil = "0"
    return (5 + dil)
